daq/panda/testing/daqTestScript.py

```diff
@@ -45,9 +45,9 @@
 #Account for 1600 ms delay in Arduino connection
 sleep(1.6)
 
-numValues = 22;
-n = 0
-while(n<50):
+num_runs = 0
+while(num_runs<50):
```

Comment:
Add spaces around the operator for good looking Python code

Reply:
done!